src/styles/containers/SiteInfo.scss

```diff
@@ -33,75 +34,87 @@
 		}
 	}
 
-	.SiteInfo_Top {
-		padding: 30px;
+	.SiteInfo_Header {
 		position: relative;
+		width: 100%;
+		max-width: 100%;
+
+		.SiteInfo_Top {
+			padding: 30px;
+			position: relative;
+			width: fit-content;
+			max-width: 100%;
+
+			.SiteInfo_SiteLinkArea {
+				margin-right: 100px;
+				a {
```

Comment:
Nit-pick: missing newline after the `margin-right`. We need to implement SASS/SCSS linting. 😃